### PR TITLE
fix(cli): minore CLI adjustments

### PR DESCRIPTION
**Pull request by zanettin**
# What is it?

- [x] Feature / enhancement
- [x] Bug
- [ ] Docs / tests

# Description

Some styling changes and improvements on the create CLI 🎉 

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/create-qwik/create-interactive.ts [hunk 1]
```diff
@@ -114,24 +124,25 @@ export async function runCreateInteractiveCli() {
...
+      s.start('Git initializing...');
+
+      try {
+        const res = [];
+        res.push(await runCommand('git', ['init'], outDir).install);
+        res.push(await runCommand('git', ['add', '-A'], outDir).install);
+        res.push(await runCommand('git', ['commit', '-m', 'Initial commit ⚡️'], outDir).install);
+
+        if (res.some((r) => r === false)) {
+          throw '';
```
Comment (zanettin) [inline packages/create-qwik/create-interactive.ts:138]: the promises above do not throw but `resolve(false)` which is not catchable. this is a workaround. if you @adamdbradley think we could safely refactor the `runCommand` fn to throw, we could get rid of these workaround 👼

File: packages/create-qwik/create-interactive.ts [hunk 2]
```diff
@@ -114,24 +124,25 @@ export async function runCreateInteractiveCli() {
 
   if (gitInitAnswer) {
     if (fs.existsSync(join(outDir, '.git'))) {
-      console.log(`Git has already been initialized`);
-      console.log(``);
-      return;
```
Comment (zanettin) [inline packages/create-qwik/create-interactive.ts:119]: atm we stop the whole CLI flow if this situation occurs which is from my point of view suboptimal.

File: packages/create-qwik/create-interactive.ts [hunk 3]
```diff
@@ -114,24 +124,25 @@ export async function runCreateInteractiveCli() {
...
-    }
-
-    s.start('Git initializing...');
-
-    try {
-      await runCommand('git', ['init'], outDir).install;
-      await runCommand('git', ['add', '-A'], outDir).install;
-      await runCommand('git', ['commit', '-m', 'Initial commit ⚡️'], outDir).install;
+      log.info(`Git has already been initialized before. Skipping...`);
```
Comment (zanettin) [inline packages/create-qwik/create-interactive.ts:127]: changed to `info` log from clack as well and updated wording a bit

File: packages/create-qwik/create-interactive.ts [hunk 4]
```diff
@@ -114,24 +124,25 @@ export async function runCreateInteractiveCli() {
...
-      s.stop('Git initialized 🎲');
-    } catch (e) {
-      s.stop('Git failed to initialize');
-      console.log(``);
-      console.log(red(`Git failed to initialize. You can do this manually by running: git init`));
-      console.log(``);
+        s.stop('Git initialized 🎲');
+      } catch (e) {
+        s.stop('Git failed to initialize');
+        log.error(red(`Git failed to initialize. You can do this manually by running: git init`));
```
Comment (zanettin) [inline packages/create-qwik/create-interactive.ts:144]: changed to `error` log from clack as well

**Comment by manucorporat** [comment]
Rocking it!

**Comment by manucorporat** [comment]
merge?

**Pull request merged**